dask_conversions/done/Dask_Coastline_Extraction_Landsat.ipynb

```diff
@@ -23,12 +23,10 @@
     "\n",
```

Comment:
<p>Although lat _extents and lat_range show the same area one is in metres the other in lat/lon. Is there a reason you changed this over? we can discuss over call if required. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SatelliteApplicationsCatapult/odc-hub/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>I didn't change it over ;) You've been tricked by the diff I'm afraid! The original notebook defined <code>query</code>  twice! The first used <code>lat_range</code>  but the second (the one which was actually used) used <code>lat_extents</code> . Check the full diff and look at this cell and the one below - you'll see what I mean.</p>

Reply:
<p>I see - that was a mistake duplication of query. Please use <strong>lat_range</strong>, and <strong>long_range</strong>, additionally please add <strong>'crs': crs</strong> to the query dictionary - see changes on master branch for reference</p>